desktop/apps/editorial_features/components/venice_2017/templates/body.jade

```diff
@@ -1,8 +1,10 @@
 .venice-body--article
   .venice-body__left
+    h2 About the Film
     +item_description(section)
     +headset_prompt
   .venice-body__right
+    h2 About the Film
```

Comment:
This is for the top part -- was included in the mobile view but not desktop on sketch, @owendodd requested adding it to desktop as well